plugins/logger/buffered.cpp

```diff
@@ -26,15 +23,15 @@
 #include <osquery/utils/system/time.h>
 #include <plugins/config/parsers/decorators.h>
 
-namespace pt = boost::property_tree;
-
 namespace osquery {
 
 FLAG(uint64,
      buffered_log_max,
      1000000,
      "Maximum number of logs in buffered output plugins (0 = unlimited)");
 
+DECLARE_bool(decorations_top_level);
```

Comment:
nit: would be nice to add `FLAG` this to our `docs`

Reply:
We have a mention of the flag in this paragraph, which exists for result logs too:
https://github.com/osquery/osquery/blob/master/docs/wiki/deployment/configuration.md#decorator-queries

I have updated the wording there to include also the status logs, but I can add the flag here also, if you meant that: https://github.com/osquery/osquery/blob/master/docs/wiki/installation/cli-flags.md

Reply:
Thanks @Smjert! Yes, I meant https://github.com/osquery/osquery/blob/master/docs/wiki/installation/cli-flags.md
I use that as a single place to look for all of the flags we have